code/modules/events/headcrabs.dm

```diff
@@ -16,15 +16,14 @@
 #define LOC_GENETICS 15
 #define LOC_ELECTRICAL 16
 #define LOC_ABANDONEDBAR 17
-#define LOC_ELECTRICAL_SHOP 18
-#define LOC_GAMBLING_DEN 19
 
 #define HEADCRAB_NORMAL 0
 #define HEADCRAB_FASTMIX 1
 #define HEADCRAB_FAST 2
 #define HEADCRAB_POISONMIX 3
 #define HEADCRAB_POISON 4
-/*#define HEADCRAB_CLOWN 5 */ //Planned for the future. maybe
+#define HEADCRAB_SPAWNER 5
+/*#define HEADCRAB_CLOWN 6 */ //Planned for the future. maybe
```

Comment:
It is generally not good to add commented-out code.

Also, for the love of all that is holy, please no clown headcrabs.